components/colorloupe/themes/spectrum.css

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@container (--system: spectrum) {
+  .spectrum-ColorLoupe {
+    --spectrum-colorloupe-express-visibility: none;
+    --spectrum-colorloupe-spectrum-visibility: block;
```

Comment:
We'll chat about this, but it's related to my comment about handling things between Spectrum & Express with custom properties rather than classes that refer to those themes.

A somewhat related example is what I did with the Menu PR based upon feedback from the SWC team. I set up [three custom properties](https://github.com/adobe/spectrum-css/pull/1508/files#diff-3b8d6209262a97e225fba580d5a675267ced29e23aadabdcc2cb73332bf11d94R165-R167), setting the ["default" one](https://github.com/adobe/spectrum-css/pull/1508/files#diff-3b8d6209262a97e225fba580d5a675267ced29e23aadabdcc2cb73332bf11d94R167) to the "shown" state, and then later, I switch between the two by resetting their definitions [here](https://github.com/adobe/spectrum-css/pull/1508/files#diff-3b8d6209262a97e225fba580d5a675267ced29e23aadabdcc2cb73332bf11d94R300) and [here](https://github.com/adobe/spectrum-css/pull/1508/files#diff-3b8d6209262a97e225fba580d5a675267ced29e23aadabdcc2cb73332bf11d94R306).